src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[20];
```

Comment:
Consider using plural form for the array name to indicate that it contains multiple figures, e.g., 'figures' instead of 'figure'.